jenkins/swift-functional-tests/10-install-ring.sh

```diff
@@ -1,12 +1,28 @@
 #!/bin/bash -xue
 
+function get_AllowEncodedSlashes {
```

Comment:
In the 0.3 branch, the driver encodes the slash here: https://github.com/scality/ScalitySproxydSwift/blob/0.3/swift_scality_backend/diskfile.py#L163